<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pong</class>
 <widget class="QMainWindow" name="Pong">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PONG: Game Setup</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Neue</family>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PONG</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="players_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PLAYERS</string>
    </property>
   </widget>
   <widget class="QLabel" name="powerups_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POWER-UPS</string>
    </property>
   </widget>
   <widget class="QLabel" name="difficulty_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AI DIFFICULTY</string>
    </property>
   </widget>
   <widget class="QLabel" name="points_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>POINTS LIMIT</string>
    </property>
   </widget>
   <widget class="QComboBox" name="players_combo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="powerups_combo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>ON</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OFF</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="difficulty_combo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>EASY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MEDIUM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HARD</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="points_combo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START GAME</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
